sbin/fireqos

```diff
@@ -63,9 +60,9 @@ shopt -s extglob
 
 FIREQOS_SYSLOG_FACILITY="daemon"
 FIREQOS_CONFIG="${FIREHOL_CONFIG_DIR}/fireqos.conf"
-FIREQOS_LOCK_FILE=/var/run/fireqos.lock
+FIREQOS_LOCK_FILE="$LOCALSTATEDIR/run/fireqos.lock"
```

Comment:
probably good to define our rundir location in autoconf and use it as we have done in netdata. also add {} to be consistent.